ui/src/album/AlbumGridView.js

```diff
@@ -53,7 +54,15 @@ const useStyles = makeStyles((theme) => ({
   },
   albumSubtitle: {
     fontSize: '12px',
-    color: theme.palette.type === 'dark' ? '#c5c5c5' : '#696969',
+    color: themeColorSelector(
+      '#696969',
+      [
+        theme.palette?.extraAttribute.theme,
+        'extradark',
+        theme.palette?.extraAttribute.subtitle,
+      ],
+      [theme.palette.type, 'dark', '#c5c5c5']
```

Comment:
Using code to select a colour is really bad (it was already bad before :) ). I think this should become a field in the `theme`, or maybe try to find one colour in the palette that is similar to this one (in the existing themes), and override in your new theme.

Reply:
OK, that seems a great way to select colours. Thanks for the suggestion I will do it.